src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -1214,6 +1216,22 @@ def getMaxPacketSize(self):
         else:
             return 32
 
+    def setMaxPacketSize(self, packetSize):
```

Comment:
Why you don't put the methods directly under the `SystemParameter` getter/setter method? - they belong there in my mind. Also it would be great, if you sort the methods to the system paramaters order:

1. storageCapacity
2. SecurityLevel
3. packetLength
4. baudRate

Also if you want, change the inconsistant naming "package size" / "package length" -> "package size" to avoid users are not confused by the different naming for the same thing.

Reply:
I put them right under other getters/setters. Yes, but this a better idea!